### ISSUE TITLE
Posting multiple document returns an empty object with silent query parameter

### ISSUE THREAD

**Issue by DiscoPYF** [arangodb/arangodb#11731]
## My Environment

* __ArangoDB Version__: ArangoDB Enterprise 3.6.3
* __Storage Engine__: MMFiles
* __Deployment Mode__: Single Server
* __Deployment Strategy__: Manual Start (Windows installation)
* __Configuration__:
* __Infrastructure__:
* __Operating System__: Windows 10
* __Total RAM in your machine__:
* __Disks in use__: SSD
* __Used Package__:

## Component, Query & Data

__Affected feature__:

Document HTTP interface.

- `POST /_api/document/{collection}#multiple` (create multiple documents)
- `DELETE /_api/document/{collection}` (delete multiple documents)

__AQL query (if applicable)__:

__AQL explain (if applicable)__:

__Dataset__:

__Size of your Dataset on disk__:

__Replication Factor & Number of Shards (Cluster only)__:

## Steps to reproduce

1. Use `POST /_api/document/{collection}?silent=true` with an array of document.

Example:

```
POST /_db/TESTDB/_api/document/TESTCOLLECTION?silent=true HTTP/1.1
Host: localhost:8529
Authorization: Basic cm9vdDpyb290
Content-Type: application/json

[{
	"testkey": "testvalue"
}]
```

Response:
```json
202 Accepted
{}
```

__Problem__:

`POST /_api/document/{collection}` returns an empty object when posting an array of document with the `silent` query parameter is set to true.
This is inconsistent because an array of document is returned in the response when `silent` is omitted or set to false.

This is also adding complexity for clients who expect an array when deserializing.
Example: https://github.com/Actify-Inc/arangodb-net-standard/issues/233

The same is happening for `DELETE /_api/document/{collection}`.
For this endpoint, the silent is not documented at all, which seems to be an oversight.

https://www.arangodb.com/docs/stable/http/document-working-with-documents.html#removes-multiple-documents

__Expected result__:

`POST /_api/document/{collection}` returns an empty array when posting an array of document with the `silent` query parameter set to true.

**Comment by DiscoPYF** [comment]
Looking back at this issue, it was probably done by design. It may not be worth changing now because it could actually break existing consumers of the API expecting an empty object.

I would be curious to hear back from the team about this though.

**Comment by Simran-B** [comment]
I agree that it's inconsistent, but as you said, we cannot change the API right now for compatibility reasons. I added it to our internal list for the 4.0 release nonetheless, where API breaking changes are possible.
